tests/test-basic.sh

```diff
@@ -24,7 +24,7 @@ set -euo pipefail
 # This test looks for specific localized strings.
 export LC_ALL=C
 
-echo "1..3"
+echo "1..41"
```

Comment:
This is kinda noisy, can we not have one test for "all ops have help?

Reply:
of course, can do that